src/main/resources/hudson/plugins/active_directory/ActiveDirectoryStatus/index.jelly

```diff
@@ -91,7 +91,7 @@
                         tr.appendChild(td2);
                         tr.appendChild(td3);
                         tr.appendChild(td4);
-                        $(demo).appendChild(tr);
+                        demo.appendChild(tr);
```

Comment:
where is the `demo` var coming from?

Reply:
Looks like there is a `<table id="demo" class="bigtable pane">` below, so it seems the intention was to do something like `document.getElementById('demo')`. However this is a pre-existing bug that existed before this PR.

Reply:
It is working (and was working before) because elements with id are automatically added to the `window` object and hence accessible as variables using their id as name https://html.spec.whatwg.org/multipage/nav-history-apis.html#named-access-on-the-window-object.

That said, there is no harm in improving the code a little now that we are here and use `document.getElementById()`. I will do the change

Reply:
Changed in #03c944f